package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "chalk": "2.4.2",
     "ejs": "3.0.2",
-    "generator-jhipster": "git://github.com/jhipster/generator-jhipster.git",
+    "generator-jhipster": "6.8.0",
```

Comment:
We need to keep this value because there are features that is not in 6.8.0 when the 6.9.0 will be release i will update this value

Reply:
with this value we can't generate a new app !

Reply:
even with --skip-checks option ?

Reply:
yes it's work with this option